spec/jobs/ama_notification_efolder_sync_job_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+describe AmaNotificationEfolderSyncJob, type: :job do
+  include ActiveJob::TestHelper
+  let(:current_user) { create(:user, roles: ["System Admin"]) }
+  let(:job) { AmaNotificationEfolderSyncJob.new }
+
+  describe "perform" do
+    before do
+      Seeds::NotificationEvents.new.seed!
+    end
+
+    # rubocop:disable Style/BlockDelimiters
+    let(:today) { Time.now.utc.iso8601 }
+    let(:appeals) {
+      create_list(:appeal, 10, :active)
+    }
```

Comment:
```suggestion
    let(:today) { Time.now.utc.iso8601 }
    let(:appeals) do
      create_list(:appeal, 10, :active)
    end
```
`let`s can accept multiline blocks with `do..end`. Using these will make it so that the `Style/BlockDelimiters` cop can remain enabled.